packages/editor/src/components/provider/index.native.js

```diff
@@ -350,8 +353,8 @@ export default compose( [
 			getEditorBlocks,
 			getEditedPostAttribute,
 			getEditedPostContent,
-		} = select( 'core/editor' );
-		const { getEditorMode } = select( 'core/edit-post' );
+		} = select( editorStore );
```

Comment:
Aha, interesting that we had to use the store objects instead of the store name literals to fix `useSelect` and `useDispatch` errors, but glad you fixed it and it helps move away from the name literals even more! 👏